extensions/pom.xml

```diff
@@ -129,8 +129,9 @@
                                 <ignoredUsedUndeclaredDependency>junit:junit::</ignoredUsedUndeclaredDependency>
                                 <ignoredUsedUndeclaredDependency>commons-logging:commons-logging::</ignoredUsedUndeclaredDependency>
 
-                                <!-- Logging related artifacts are exposed by the runtime -->
+                                <!-- Logging related artifacts are exposed by the BOOTSTRAP ClassLoader -->
                                 <ignoredUsedUndeclaredDependency>org.slf4j:slf4j-api:jar:</ignoredUsedUndeclaredDependency>
+                                <ignoredUsedUndeclaredDependency>javax.inject:javax.inject::</ignoredUsedUndeclaredDependency>
```

Comment:
No, it is too bad that these properties cannot be have also the scope so it would be more easy to exclude all test dependencies with just one rule. But is optional so let me remove the unnecessary segment from those patterns.

```
List of dependencies that will be ignored if they are used but undeclared. The filter syntax is:
[groupId]:[artifactId]:[type]:[version]
where each pattern segment is optional and supports full and partial * wildcards. An empty pattern segment is treated as an implicit wildcard. *
For example, org.apache.* will match all artifacts whose group id starts with org.apache., and :::*-SNAPSHOT will match all snapshot artifacts.
```